test/e2e/upg_e2e.go

```diff
@@ -1477,6 +1481,40 @@ func describeGTPProxy(title string, ipMode framework.UPGIPMode) {
 	})
 }
 
+func describeNAT(f *framework.Framework) {
+	ginkgo.Describe("NAT translations", func() {
+		ginkgo.BeforeEach(func() {
+			f.VPP.Ctl("nat44 enable sessions 1000 endpoint-dependent")
+			f.VPP.Ctl("set interface nat44 in upf-nwi-sgi out host-sgi0")
+			f.VPP.Ctl("upf nat pool 144.0.0.20 - 144.0.0.120 block_size 512 nwi sgi name testing")
+			f.VPP.Ctl("nat44 controlled enable")
+		})
+
+		verify := func(sessionCfg framework.SessionConfig) {
+			sessionCfg.NatPoolName = "testing"
+			seid := startMeasurementSession(f, &sessionCfg)
+			trafficCfg := smallVolumeHTTPConfig(nil)
+			trafficCfg.ExpectedAddr = "144.0.0.20:10128"
+			trafficRec := &traffic.PreciseTrafficRec{}
+			runTrafficGen(f, trafficCfg, trafficRec)
+			foundAddr := trafficRec.ShowClientAddr()
+			gomega.Expect(foundAddr).To(gomega.BeEquivalentTo(trafficCfg.ExpectedAddr))
```

Comment:
perhaps not using `trafficCfg.ExpectedAddr` at all and just doing `gomega.Expect(trafficRec.ClientAddr()).To(gomega.BeEquivalentTo("144.0.0.30:55555"))` should be enough

Reply:
done